homeassistant/components/thermostat/radiotherm.py

```diff
@@ -0,0 +1,127 @@
+"""
+homeassistant.components.thermostat.radiotherm
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adds support for Radio Thermostat wifi-enabled home thermostats
+"""
+import logging
+import datetime
+from urllib.error import URLError
+
+from homeassistant.components.thermostat import (ThermostatDevice, STATE_COOL,
+                                                 STATE_IDLE, STATE_HEAT)
+from homeassistant.const import (CONF_HOST, CONF_NAME, TEMP_FAHRENHEIT)
+
+REQUIREMENTS = ['radiotherm==1.2']
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Sets up the Radio Thermostat. """
+    logger = logging.getLogger(__name__)
+
+    try:
+        import radiotherm
+    except ImportError:
+        logger.exception(
+            "Error while importing dependency radiotherm. "
+            "Did you maybe not install the radiotherm dependency?")
+        return
+
+    # Detect hosts with hass discovery, config or radiotherm discovery
+    hosts = []
+    if discovery_info:
+        logger.info('hass radiotherm discovery', discovery_info)
```

Comment:
There is no support right now for discovery of radiotherm, so you can leave this out. (what can be discovered depends on [netdisco](https://github.com/balloob/netdisco))